docs/design.html

```diff
@@ -610,10 +610,12 @@ <h4><a id="design_quotasenforcement" href="#design_quotasenforcement">Enforcemen
         having a fixed cluster wide bandwidth per client because that would require a mechanism to share client quota usage among all the brokers. This can be harder to get right than the quota implementation itself!
     </p>
     <p>
-        How does a broker react when it detects a quota violation? In our solution, the broker does not return an error rather it attempts to slow down a client exceeding its quota.
-        It computes the amount of delay needed to bring a guilty client under its quota and delays the response for that time. This approach keeps the quota violation transparent to clients
-        (outside of client-side metrics). This also keeps them from having to implement any special backoff and retry behavior which can get tricky. In fact, bad client behavior (retry without backoff)
-        can exacerbate the very problem quotas are trying to solve.
+        How does a broker react when it detects a quota violation? In our solution, the broker first computes the amount of delay needed to bring a client exceeding its quota and
+        returns a response with the delay immediately. In case of a fetch request, the response will not contain any data. Then, the broker mutes the channel to the violating client,
+        not to process requests from the client anymore, until the delay is over. Upon receiving a response with a non-zero delay duration, the Kafka client layer will also refrain from
+        sending further requests to the broker during the delay. This approach keeps the quota violation transparent to clients (outside of client-side metrics). This also keeps them
```

Comment:
We probably should change or remove the sentence that says `This approach keeps the quota violation transparent to clients (outside of client-side metrics). This also keeps them
from having to implement any special backoff and retry behavior which can get tricky. In fact, bad client behavior (retry without backoff) can exacerbate the very problem quotas are trying to solve`.